### PR TITLE
Version 1.3.0

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Reset the build number as well.

**Comment by jakirkham** [comment]
Please let me know if the change I pushed to your branch is ok @jjhelmus.

**Comment by jjhelmus** [comment]
LGTM.

**Comment by jakirkham** [comment]
Thanks @jjhelmus.

**Pull request merged**